### DISCUSSION TITLE
Button Group Release Prioritization

### DISCUSSION THREAD

**Discussion by GiantRobots** [twilio-labs/paste#1211] [General] [answered]
Hey all, we use button groups pretty frequently in our designs at SendGrid and I see they're on the roadmap for October of this year. Is there a voting mechanism or something else to get that re-prioritized for an earlier release?

You can see what we're doing in the meantime which isn't great looking or particularly accessible. 
https://codesandbox.io/s/paste-starter-kit-forked-w1071?file=/src/index.tsx

**Comment by gloriliale** [comment, marked as answer]
Hi @GiantRobots! It looks like we haven't done any roadmap review (where we make these kinds of prioritization calls) w/ sendgrid teams, so let's fix that. Who are the product designer and product manager contacts we could get in touch w/ for setting that up?

**Reply by GiantRobots** [reply]
Sean Mateer is our designer

George Chi - PM
Scott Kawaii - PM
